spec/aged_brie_spec.js

```diff
@@ -1,52 +1,57 @@
 describe("Aged Brie", function() {
+    const anonymousName = "Aged Brie";
+    const anonymousSellIn = 5;
+
     beforeEach(function() {
-        brie1 = new AgedBrie("Aged Brie", 4, 2);
-        brie2 = new AgedBrie("Aged Brie", 4, 50);
-        gildedRose = new Inn([brie1, brie2]);
+        BRIE_WITH_SELLIN_4_DAYS = new AgedBrie("sellIn_4", 4, 2);
+        BRIE_WITH_QUALITY_50 = new AgedBrie("quality_50", 4, 50);
+        gildedRose = new Inn([BRIE_WITH_SELLIN_4_DAYS, BRIE_WITH_QUALITY_50]);
     });
 
     describe("Aged Brie creation", function() {
        it("can be created with a name", function() {
-          expect(brie1.name).toEqual("Aged Brie");
+          expect(BRIE_WITH_SELLIN_4_DAYS.name).toEqual("sellIn_4");
        });
 
        it("can be created with sell-in", function() {
-          expect(brie1.sellIn).toEqual(4);
+          expect(BRIE_WITH_SELLIN_4_DAYS.sellIn).toEqual(4);
        });
 
        it("can be created with quality", function() {
-           expect(brie1.quality).toEqual(2);
+           expect(BRIE_WITH_SELLIN_4_DAYS.quality).toEqual(2);
        });
 
        it("can be created with a max quality of 50", function() {
            expect(function() {
-               new AgedBrie("brie4", 5, 51)
-           }).toThrow(new Error("Max quality is 50."));
+               new AgedBrie(anonymousName, anonymousSellIn, 51)
+           }).toThrow(new Error("Quality must be within 0 - 50 inclusive."));
```

Comment:
If you define your own error type (e.g. `QualityError`), you need not duplicate this error-creation code between the two tests. Additionally, the tests could read very nicely as `...toThrow(QualityError)` or `...toThrow(new QualityError())` depending on whether or not you want to use a constant; it doesn't make much difference either way, I think.